### PR TITLE
Eit updates

### PR DESCRIPTION
**Pull request by gfto**

### CODE REVIEW

File: demux.c [hunk 1]
```diff
@@ -3009,23 +3032,27 @@ static void HandleEIT( uint16_t i_pid, uint8_t *p_eit, mtime_t i_dts )
...
+        p_sid->eit_table[eit_table_id].data[i_section] = p_eit;
         goto out_eit;
     }
 
-    free(p_sid->pp_eit_sections[i_section]);
-    p_sid->pp_eit_sections[i_section] = p_eit;
+    free(p_sid->eit_table[eit_table_id].data[i_section]);
+    p_sid->eit_table[eit_table_id].data[i_section] = p_eit;
 
     if ( b_print_enabled )
```
Comment (cmassiot) [inline demux.c]: Please add a new option to enable/disable printing the EPG to stdout. I am afraid such a high-volume data may flood stdout.

**Review by cmassiot — CHANGES_REQUESTED** [review]
Except b_print_enabled, LGTM

**Comment by cmassiot** [comment]
Ping

**Comment by gfto** [comment]
I'll resubmit the pull with the required fixes.

**Pull request closed without merging**